docs/source/en/api/schedulers/overview.mdx

```diff
@@ -83,4 +83,24 @@ The class [`SchedulerOutput`] contains the outputs from any schedulers `step(...
 
 ### KarrasDiffusionSchedulers
 
+`KarrasDiffusionSchedulers` encompasses the main generalization of schedulers in Diffusers. The schedulers in this class are distinguished, at a high level, by their noise sampling strategy; the type of network and scaling; and finally the training strategy or how the loss is weighed.
+
+The different schedulers, depending on the type of ODE solver, fall into the above taxonomy and provide a good abstraction for the design of the main schedulers implemented in Diffusers. The schedulers in this class are given by the following `Enum`:
+
+```python
+class KarrasDiffusionSchedulers(Enum):
```

Comment:
Hmm, ideally I think the docs should be generated directly from the code so that we don't have to manually update the docs every time we add a scheduler